core/controllers/download.py

```diff
@@ -106,9 +106,12 @@ def sort_output_dataframes(df: pd.DataFrame, sheet: str) -> pd.DataFrame:
     :param df: The DataFrame to be sorted.
     :param sheet: The name of the Excel output sheet.
     :return: Sorted dataframe.
+    :raise: ValueError: if "sheet" is not present in TABLE_SORT_ORDERS configuration
     """
-
-    df.sort_values(TABLE_SORT_ORDERS.get(sheet), inplace=True)
+    if sort_order := TABLE_SORT_ORDERS.get(sheet):
+        df.sort_values(sort_order, inplace=True)
+    else:  # Programmer error - please add sheet to TABLE_SORT_ORDERS
+        raise ValueError(f"No table sort order defined for table extract: {sheet}.")
```

Comment:
Do we want to add the comment to the exception message?
e.g.
`No table sort order defined for table extract: {sheet}. lease add sheet to TABLE_SORT_ORDERS"`
To save people having to hunt through the code?